github_pipeline_observability/README.md

```diff
@@ -0,0 +1,179 @@
+    * Edit OpenPipeline configurations: `openpipeline:configurations:write`
+    * Create and edit documents: `document:documents:write`
+    * View documents: `document:documents:read`
+
+3. Store the retrieved client ID, secret, and token endpoint as an environment variable.
+<!-- windows version -->
+```
+$env:OAUTH_CLIENT_ID='<YOUR_CLIENT_ID>'
+$env:OAUTH_CLIENT_SECRET='<YOUR_CLIENT_SECRET>'
+$env:OAUTH_TOKEN_ENDPOINT='https://sso.dynatrace.com/sso/oauth2/token'
+```
+<!-- linux / macOS version -->
+```
+export OAUTH_CLIENT_ID='<YOUR_CLIENT_ID>'
+export OAUTH_CLIENT_SECRET='<YOUR_CLIENT_SECRET>'
+export OAUTH_TOKEN_ENDPOINT='https://sso.dynatrace.com/sso/oauth2/token'
+```
+
+4. Clone the [Dynatrace configuration as code sample](https://github.com/Dynatrace/dynatrace-configuration-as-code-samples) repository and go to `github_pipeline_observability`.
+```
+git clone https://github.com/Dynatrace/dynatrace-configuration-as-code-samples.git
+cd github_pipeline_observability
+```
+
+5. Edit the `manifest.yaml` by exchanging the `<YOUR-DT-ENV-ID>` placeholder with your Dynatrace environment ID.
+```
+manifestVersion: 1.0
+projects:
+  - name: pipeline_observability
+environmentGroups:
+  - name: group
+    environments:
+      - name: <YOUR-DT-ENV-ID>
+        url:
+          type: value
+          value: https://<YOUR-DT-ENV-ID>.apps.dynatrace.com
+        auth:
+            oAuth:
+              clientId:
+                name: OAUTH_CLIENT_ID
+              clientSecret:
+                name: OAUTH_CLIENT_SECRET
+              tokenEndpoint:
+                type: environment
+                value: OAUTH_TOKEN_ENDPOINT
+```
+
+## Steps 
+
+### 1. Configure Dynatrace using Monaco
+
+In this section, you will upload two Dashboards and configure the ingest endpoint for GitHub Webhooks in Dynatrace. Before you continue, please check your OpenPipeline configuration for *Software development lifecycle events*.
+
+1. In Dynatrace, navigate to **OpenPipeline**.
+2. Expand **Events** and click **Software development lifecycle**.
+3. Open **Ingest source** and check if another ingest source exists except the built-in *Endpoint for Software Development*.
+4. Open **Dynamic routing** and check if another route exists excpet the *Default route*.
+5. Open **Pipelines** and check if another pipeline exists except the built-in *events.sdlc*. 
+6. If you  have the default and built-in configuration, you can run Monaco to deploy the provided configuration; otherwise, you must merge the configuration file before updating it. 
+
+#### Run Monaco deploy
+
+Run the following command to apply the provided configuration. 
+
+```
+monaco deploy manifest.yaml
+```
+
+#### Merge configuration before running Monaco deploy
+
+1. Download your OpenPipeline configuration.
+```
+monaco download
+```
+
+2. Merge the content of `download_<DATE>_<NUMBER>\project\openpipline\events.sdlc.json` into the file `events.sdlc.github.json`.
```

Comment:
Are you using \ on purpose here? Shouldn't this be 
```suggestion
2. Merge the content of `download_<DATE>_<NUMBER>/project/openpipline/events.sdlc.json` into the file `events.sdlc.github.json`.
```

Reply:
My mistake. Fixed it.